format/view-spec.md

```diff
@@ -42,12 +42,27 @@ An atomic swap of one view metadata file for another provides the basis for maki
 
 Writers create view metadata files optimistically, assuming that the current metadata location will not be changed before the writer's commit. Once a writer has created an update, it commits by swapping the view's metadata file pointer from the base location to the new location.
 
+### Materialized Views
+
+Materialized views are a type of view that precompute the data from the view query.
+When queried, materialized views return the precomputed data, shifting the cost of query execution to the precomputation step.
+
+Iceberg materialized views are implemented as a combination of an Iceberg view and an underlying Iceberg table, known as the storage table, which stores the precomputed data.
+The metadata for a materialized view extends the common view metadata, adding a pointer to the precomputed data and information about the refresh state to determine if the data is still fresh. 
+The refresh state is composed of data about the so-called "source tables", which are the tables referenced in the query definition of the materialized view. 
```

Comment:
Nit: how about new paragraph for 'refresh state'

and then bullet point for each state

Also , is refresh state the same as storage table state?  Its a bit confusing to read as is, can use the same terminology if its the same?